thienhophu/ProjectTracker
# Add create new project form validation

**Why we have this PR:**
- Create a new project form that needs validation if the user lets it blank

**How to test:**
- Create a new project while letting Project's name or Project's description blank to see if it required to be input

**Tested on:**
